Author: Ole Streicher <olebole@debian.org>
Description: Propagate LDFLAGS
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,7 +59,7 @@
 #set( CASA_assay ${CMAKE_SOURCE_DIR}/install/assay )
 
 
-set( CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined")
+#set( CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined")
 
 set( dbusxx dbus-c++ )
 set( casa_packages /usr/lib64/casapy )
@@ -162,12 +162,12 @@
     set( _deps X11 FORTRAN )
   endif()
   if (CMAKE_SYSTEM MATCHES ^Darwin-11 AND NOT LLVMCOMPILER )
-     set( CMAKE_SHARED_LINKER_FLAGS "-Wl,-undefined -Wl,error -framework AppKit -lgfortran" )
+     set( CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-undefined -Wl,error -framework AppKit -lgfortran" )
   else()
      if (APPLE) 
-        set( CMAKE_SHARED_LINKER_FLAGS "-Wl,-undefined -Wl,error -framework AppKit" )
+        set( CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-undefined -Wl,error -framework AppKit" )
      else()
-        set( CMAKE_SHARED_LINKER_FLAGS "-Wl,-undefined -Wl,error" )
+        set( CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-undefined -Wl,error" )
      endif()
   endif()
   casa_find( PGPLOT
